pkg/pillar/cmd/domainmgr/domainmgr.go

```diff
@@ -2098,14 +2106,13 @@ func DomainCreate(status types.DomainStatus) (int, string, error) {
 
 // Launch app/container thru rkt
 // returns domainID, podUUID and error
-func rktRun(domainName, xenCfgFilename, imageHash string) (int, string, error) {
+func rktRun(domainName, xenCfgFilename, imageHash string, envList map[string]string) (int, string, error) {
```

Comment:
nit - envList --> List sounds like its an array - I guess a hangover from python..

envVars or envVarMap would be better names